packages/grafana-data/src/types/dataLink.ts

```diff
@@ -52,6 +52,11 @@ export interface DataLink<T extends DataQuery = any> {
 
   origin?: DataLinkConfigOrigin;
   sortIndex?: number;
+
```

Comment:
cc @leeoniya

Reply:
if you're asking about `sortIndex` above, it will go away it was accidentally left over from a recent PR that removed its use.

were you asking for feedback about the `meta` additions below? :thinking: